modules/msk/genotypevariants/all/main.nf

```diff
@@ -0,0 +1,62 @@
+process GENOTYPEVARIANTS_ALL {
+    tag "$meta.id"
+    label 'process_single'
+
+
+    conda "${moduleDir}/environment.yml"
+    container "${ workflow.containerEngine == 'singularity' && !task.ext.singularity_pull_docker_container ?
+        'ghcr.io/msk-access/genotype_variants:sha-7e2c3c8a':
```

Comment:
Should we add the released genotype_variants docker image before merging this PR into traceback?